source/Halibut/Transport/PollingClient.cs

```diff
@@ -13,47 +15,66 @@ public class PollingClient : IPollingClient
         readonly ILog log;
         readonly ISecureClient secureClient;
         readonly Uri subscription;
-        readonly Thread thread;
-        readonly CancellationToken cancellationToken;
+        Thread? pollingClientLoopThread;
         bool working;
-        readonly Func<RetryPolicy> createRetryPolicy;
-
-        [Obsolete("Use the overload that provides a logger. This remains for backwards compatibility.")]
-        public PollingClient(Uri subscription, ISecureClient secureClient, Func<RequestMessage, ResponseMessage> handleIncomingRequest, Func<RetryPolicy> createRetryPolicy)
```

Comment:
No one within halibut uses these and no one outside of halibut would be using this so it should be safe to rmove.